lang/po/new/cs.po

```diff
@@ -823,7 +820,7 @@ msgstr "MMU2 pripojeno"
 # MSG_SELFTEST_MOTOR c=18
 #: messages.c:96
 msgid "Motor"
-msgstr ""
+msgstr "Motor"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.